io_scene_godot/__init__.py

```diff
@@ -218,12 +219,14 @@ def execute(self, context):
                 "filter_glob",
                 "xna_validate",
             ))
-
+            logging.getLogger().addHandler(exporter_log_handler)
             return export_godot.save(self, context, **keywords)
         except ValidationError as error:
             self.report({'ERROR'}, str(error))
             return {'CANCELLED'}
-
+        finally:
+            if exporter_log_handler:
+                logging.getLogger().removeHandler(exporter_log_handler)
 
```

Comment:
> io_scene_godot/__init__.py:231:1: E302 expected 2 blank lines, found 1

The style check is [failing](https://travis-ci.org/github/godotengine/godot-blender-exporter/jobs/758363096#L1048) because pep8 wants two blank lines between functions.

Reply:
yes, of course, will be soon